golang pg test

rekby opened the pull request:
### Changelog entry <!-- a user-readable short description of changes introduced in this PR -->

...

### Changelog category <!-- remove all except one -->

* Not for changelog (changelog entry is not required)

### Additional information

...

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Update contrib/restricted/boost/move to 1.86.0
b1ffaaff544bd1ab1af72314e524d49221a5f67e

tatakir committed:
Save peers and direct peers to dm cache
3ca03f9795edc451ae7c7513f2869c3d0cfb0ff2

Commit:
Add another part SDC contrib to allow NO_LINT

Еще одна пачка проектных контрибов.
53591f2d7360aa45b12fa2215945c9fc3569df19

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Intermediate changes

achulkov2 committed:
Add dynamic table log writer

Adds a new `dynamic_table` log writer, which writes log events to the configured sorted or ordered dynamic table.
Log events are serialized into a YSON representation, which is interpreted similarly to the `insert-rows` cli command.
It is your responsibility to create the table with a compatible schema.
See integration test for example configuration.

The writer is available for use in all YT server components.

TRIVIAL

---
bfb7bf098156a8b16a6fdbf018c54858b1479b03

Pull Request resolved: https://github.com/ytsaurus/ytsaurus/pull/631

Co-authored-by: eshcherbin <eshcherbin@yandex-team.com>

robot-piglet committed:
Intermediate changes

Commit:
Track job CPU utilization on Flow workers
8fc3af8a8fe764c4c60363a67733aedeb70c9226

VPolka committed:
return implicit query params type into feature flag (#8065)

robot-piglet committed:
Intermediate changes

GrigoriyPA committed:
YQ-3363 fix internal error for insert without params (#8074)

Commit:
External build system generator release 98

Update tools: yexport, os-yexport
c6e64241d2cbdcf9e95017a87c0e3959e88db79d

kruall committed:
Change activation queue to variant of queues (#8078)

Commit:
YT-21838: Introduce nested aggregate columns
6bac1e5898fd226d2152bf14df7e1a2b227895aa

Commit:
YT-21993: Allow to omit including `type-def.h` in header files that use Phoenix v2
02dce8a147c6036c91d4c69dac40e9a9993fc26d

Commit:
YT-21993: Add forgotten `override` specifiers
b77c55a40214fce1a37af17fb6d9bf7e04805eef

vitstn committed:
Fixes #8019 (#8109)

vporyadke committed:
improve failsafe on cuttting history (#6918)

rekby committed:
build image

rekby committed:
style

Commit:
YT-21993: Allow to mix Phoenix v1 and v2 classes in the hierarchy
da8e97a3a6b4fdacb65a37420ccb2ec916ab604b

robot-piglet committed:
Update contrib/restricted/boost/bind to 1.86.0
36e5db27fff0e5e63069b8863a31054cd7e5b443

niksaveliev committed:
Fix describe consumer (#8095)

ivanmorozov333 committed:
multi heads allocations manager (#8126)

Commit:
Enable fix-elf only with _FIX_ELF flag
f1b4ab8d00e0ac2175f0ea34a0353ce2963549b7

Pseudolukian committed:
Fix YQL optional types note in YT DOC (#8057)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

SloNN committed:
pg (#7381)

Co-authored-by: Alex Dmitriev <slonnn@yandex-team.ru>
Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

zinal committed:
Moved cluster topology information to the topology overview section (#7144)

Co-authored-by: Ivan Blinkov <ivan@ydb.tech>

blinkov committed:
[docs] update create_table.md (#8071)

robot-piglet committed:
Intermediate changes

dimdim1177 committed:
Fix headers only library templates

Fix headers only libraries
8d98d2115e9207d233331bf0ab2ba307d84b255c

swalrus1 committed:
Add unit-test for data eviction in tiering (#7971)

Commit:
Update github.com/ydb-platform/fq-connector-go to 0.5.5-rc.1 (#8127)

maximyurchuk commented on ydb/tests/postgres_integrations/library/ya.make:
```diff
@@ -0,0 +1,22 @@
+#    ydb/public/sdk/python
+)
+
+REQUIREMENTS(ram:10)
```
Вот так не стоит писать, ram не учитывается в локальных запусках ya make, но неявно выставляет требование по cpu

azevaykin committed:
Statistics: AnalyzeStatus should use the same sender to preserve event order (#8136)

robot-piglet committed:
Intermediate changes

robot-piglet committed:
Update contrib/restricted/boost/predef to 1.86.0
a86177b405d83df3dba0a9c13612ef7144cdab7b

robot-piglet committed:
Update contrib/restricted/boost/container to 1.86.0
1d9f36e4b2e11f3aabe2d7350166df08c1cc72d1

snaury committed:
Show a nicer demangled name in block/unblock messages (#8138)

maximyurchuk committed:
Add docs dir into RECURSE ydb (#8092)

robot-piglet committed:
Intermediate changes

Commit:
YT-20983: Add dynamic tables versioned map reduce read
12c8b4a9cd18c32a89bb2bc6d78fd4dc0f1da7b5

dorooleg committed:
streaming grace join test (#3862)

nikitka committed:
ci: download the latest github runner on a VM start (#7987)

azevaykin committed:
Debug logging in WaiFor (#8137)

ivanmorozov333 committed:
correct portions index usage for control memory scan intervals (#8135)

swalrus1 committed:
Don't skip null values on COUNT(*) in column shards (#8024)

snaury committed:
Use granular updates for mediator timecast when available (#8090)

vitstn committed:
Update postgis functions from tests (#8133)

kruall committed:
Add UseRingQueue to ActorSystemConfig (#8122)

SammyVimes committed:
CMS editor async load fix (#8140)

Loading editor.js doesn't block the rest of the page's content loading anymore

fexolm committed:
Add scan/insert while alter test (#8094)

azevaykin committed:
Statistics: SendBaseStatsToSA return delta for schedule next event (#8119)

swalrus1 committed:
Add column shard Controller parameters to AppData (#8000)

StekPerepolnen committed:
add http-handle returning configs (#7830)

kungasc committed:
Apply old Resource Broker config (#8087)

qrort committed:
pass db path to pg_tables instead of tenant name. Resolves #7616 (#8061)

pavelvelikhov committed:
Lowered threshold for MapJoin (#8147)

robot-piglet committed:
Intermediate changes

Commit:
YT-21993: Fix duplicate type tag on derive from template
7ec29a7b370f343f910075ff9fd87ea9469d2aa0

p4vook committed:
GLOBAL OTBIVKA FIX ୧༼ಠ益ಠ༽୨

Make all "otbivka" exactly 80 symbols.

Command:
```bash
sed -i 's#^//////////////\+$#////////////////////////////////////////////////////////////////////////////////#g' $(find yt \( -name '*.cpp' -or -name '*.h' -or -name '*.proto' \) -not -path 'yt/yt/experiments/private/tabletvis/imgui/*')
```
30a0ba9f42ba1909d3cee11fe12bba14c7c8b039

Krisha11 committed:
Fix skiff usage in dq (#7864)

vitstn committed:
Fixed validation of lambda before apply (#8153)

lll-phill-lll committed:
added flamegraph to github
03edf63f51b2878f5e14d39a19a3e1aa72de5398

vporyadke committed:
actually tie per-dc-followers to dc (#8069)

Co-authored-by: Aleksandr Zalialov <zalyalov@mr-nvme-testing-01.search.yandex.net>

vporyadke committed:
treat initial vdisk status as ok (#8150)

igormunkin committed:
Refactor tests for BlockMapJoinCore computation node (#8129)

Enjection committed:
Fix config resolver in ui (#8157)

yumkam committed:
tpc ds: manually optimize join order to workaround #7565 (#8163)

StekPerepolnen committed:
query float test asan (#8161)

nikitka committed:
ci: fix logic in check-is-mergeable check (#8152)

nikitka committed:
ci: upgrade actions/checkout@v4, actions/github-script@v7, add test_retry_count input for build_and_test_ya action (#8164)

Commit:
YT-22634: Add code for total memory limit exceeded alert
2d3bed035fb10346b6295993fe1f08f714411edd

azevaykin committed:
Statistics: disable some internal events without feature flag (#8159)

Krisha11 committed:
YQLAgent, free progress memory (#7585)

Commit:
Do not use YQL_LOG during YQL plugin initialization (#6798)

mxkovalev committed:
YQL: Fuse reduces with presort settings (#8117)

alexvru committed:
Report UNKNOWN status to SysView for disks that have not checked in yet (#8144)

eivanov89 committed:
use new avx macros (#7494)

va-kuznecov committed:
PDisk LWTracing improvements (#8160)

Co-authored-by: Vlad Kuznecov <va-kuznecov@nebius.com>

Alek5andr-Kotov committed:
The value of the `WriteInflightSize` in the main partition (#8116) (#8142)

dimdim1177 committed:
Macro USE_ANNOTATION_PROCESSOR

Macro USE_ANNOTATION_PROCESSOR
670d47e64a48e59e9b190875a9729bc9e9921e15

dimdim1177 committed:
Add sem use_annotation_processor

Add sem use_annotation_processor
f5ffdd48bd9f14fc3bf52b8f4a22d0b9277305a2

maximyurchuk committed:
Do docker-compose tests in parallel (#8165)

Commit:
YT-22435: Using anonymous memory limit in tcmalloc
93d6930ce24f343ccd23c55e7264db80f2e0046b

Commit:
Update github.com/ydb-platform/fq-connector-go to 0.5.5-rc.2 (#8183)

shadchin committed:
Fix DeprecationWarning

Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.

https://docs.python.org/3/library/tarfile.html#tarfile.TarFile.extractall
79005c43af0996c47603d56b5663378c89fa9f2e

shadchin committed:
Switch on platform_dispatchers
93438876ebd102c07d52ce42cc48e5465a7277ff

maximyurchuk committed:
Revert "ci: upgrade actions/checkout@v4, actions/github-script@v7, add test_retry_count input for build_and_test_ya action" (#8185)

Commit:
New version of the tld SKIP_CHECK SKIP_REVIEW
4fbdf2c6326bb76d9e4f3f771383bc2b00a8d3e2

robot-piglet committed:
Intermediate changes

Commit:
YT-25593: Replace TString with std::string for addresses, hostnames etc
e10316e900a0dd13e98e68dacd3338f5d277c8a2

zaverden committed:
feat(conf): set requirements for build to cpu:2
c321682fac0742cd7c6fd860085140989c4729e8

pashandor789 committed:
[KQP] ANALYZE retries has been added. (#8115)

Commit:
YT-22619: Improve merge job memory tracking
38867a055f64fbf74a008e2c1dd1ef1cda73736d

robot-piglet committed:
Intermediate changes

abogutskiy committed:
Fix clang logical-op-parentheses warning

No description

---
687cd74eb85cc26a02a9e2536693ad9c9d710416

Pull Request resolved: https://github.com/ytsaurus/ytsaurus/pull/795

va-kuznecov committed:
Fix counter names in group_write load actor (#8058)

E1pp committed:
YT-22307: Empty template for dist write api
e9554dea04d2a2fb0c9d8cbee012afa88382e715

Commit:
Automatic release build for test_tool, os_ya, ya_bin, os_test_tool

From hash: [8de4949957d3a7517bb29f7ce6af1d4eec61afea](https://a.yandex-team.ru/arcadia/commit/8de4949957d3a7517bb29f7ce6af1d4eec61afea)
From revision: [14696290](https://a.yandex-team.ru/arcadia/commit/rXXXXXX)
[CI flow](https://a.yandex-team.ru/projects/ya_make/ci/releases/flow?dir=devtools%2Fya&id=release-ya-bin2-ya-bin3-tts&version=463)
Flow triggered by user: [robot-ci](https://staff.yandex-team.ru/robot-ci)

Update tools: test_tool, os_ya, ya_bin, os_test_tool
e8fd052c7e045bbfc311a349b246422dbe698ae4

aavdonkin committed:
Added sparsed columns tests for standalone tables (#8167)

vitstn committed:
Removed weird restriction for optional pg (#8175)

Commit:
Temporary enable fix-elf by default
bd9670ad3d9e38289bae7d361cdb0923077bb552

robot-piglet committed:
Intermediate changes

azevaykin committed:
Statistics: retry AnalyzeTable to ColumnShard (#8190)

abyss7 committed:
Wait for all CAs inside Executer before shutdown (#7829)

evanevanevanevannnn committed:
inference json support (#8102)

lll-phill-lll committed:
Try adding flamegraph with simple copy
6d3bfb2f6ddb8ad122ad37485ce2d0ac8346e0ee

alexv-smirnov committed:
build: refresh Embedded UI (v6.16.0) (#8143)

Co-authored-by: astandrik <astandrik@users.noreply.github.com>

azevaykin committed:
Statistics: Remove IsSchemeshardSeen, rely on persisted ScheduleTraversals (#8182)

andrewstalin committed:
Statistics: Expanded ForceTraversalOperations scheme, add CreatedAt (#8194)

alexv-smirnov committed:
Import libraries 240823-1155

rvu1024 committed:
[yt provider] Allow append with same column groups (#8176)

loochek committed:
Fix RebuildCalcOverWindowGroup failing with PercentRank/CumeDist/Ntile (#8148)

molotkov-and committed:
OIDC: Add support redirects (#8151)

siarheivesialou committed:
LOGBROKER 8891 fix parsing queue url (#8203)

swalrus1 committed:
Forbid creating empty tiering rules in column shards (#8154)

alexd65536 committed:
set partitioning settings for statistics table (#8193)

azevaykin committed:
Statistics: reset events in test observers (#8118)

gridnevvvit committed:
subscribe on changes in resource broker (#8187)

MBkkt committed:
Refactor build index (#7658)

robdrynkin committed:
Dsproxy put lwtrack (#8168)

dcherednik committed:
Set EnableUniqConstraint by default (#8180)

ildar-khisambeev committed:
count-min-sketch as index (#7978)

Tony-Romanov committed:
Add stat counters for PutsPerSecond. (#8201)

alexvru committed:
Support STLOG json format for JSON output (#8205)

kungasc committed:
Always limit HardLimitBytes with MemTotal (#8223)

rvu1024 committed:
Handle Write in DQ recapture (#8212)

ivanmorozov333 committed:
dont create temporary sparsed array on merging (#8174)

fedor-miron committed:
Block datetime2 (#6472)

GrigoriyPA committed:
YQ-3570 added s3 wildcards validations (#8017)

alexvru committed:
Fix duplicated responses in collect/propose query (#8249)

Hor911 committed:
Check incomplete refs and dedup stats in case of failures (#8234)

Hor911 committed:
Fix replicated stage CTE refs between precomputes (#7788)

resetius committed:
Add decimal unary kernels (#8254)

abyss7 committed:
Do not fill tables for prepared query on compilation failure (#8210)

nikvas0 committed:
Allow single CTAS without perstatement (#8241)

nikvas0 committed:
Allow sinks for data query (#8242)

pixcc committed:
Add node name to labels (#8237)

maximyurchuk committed:
Upload html summary before updating status (#8224)

nikitka committed:
ci: upgrade actions/checkout@v4, actions/github-script@v7, test_retry_count input variable (#8216)

ildar-khisambeev committed:
Revert add status PROCESSING to statistics proto (#8253)

shnikd committed:
PgWire auth with ApiKey (#7973)

shnikd committed:
Fix path type for pg_class (#8243)

Hor911 committed:
Aggregated spilling stats in KQP pipeline (#8220)

adameat committed:
optimize nodes handler (#8111)

GrigoriyPA committed:
YQ WM fixed databse checking (#8251)

pixcc committed:
Fix overriding feature flags in CMS (#8238)

GrigoriyPA committed:
YQ-3566 fix sql injection in create binding request (#7984)

azevaykin committed:
Statistic: Delete analyze after deadline (#8214)

dorooleg committed:
listing fix (#7643)

SammyVimes committed:
Fix LWTrace HTML page generation (#8158)

rvu1024 committed:
Fix PoolTrees, TentativePoolTrees pragmas (#8272)

CyberROFL committed:
Async replication: configurable default retention period (#8270)

CyberROFL committed:
Allow to alter cdc topic's retention period (#8264)

igormunkin committed:
Introduce computation node for BlockMapJoinCore (Left and Inner) (#8030)

MrLolthe1st committed:
Fix TPogram fields order (#8268)

ulya-sidorina committed:
fix(kqp): always take snapshot for queries with stream lookup (#8267)

vitstn committed:
visit & hash evaluation key in the one pass (#8277)

yumkam committed:
tpc ds q32: fix yql translation (#8295)

azevaykin committed:
Statistics: more logs in statistics service (#8282)

yumkam committed:
tpc ds q14: temporary workaround CBO bug (#8298)

MrLolthe1st committed:
Fix arrow yt column converter (#8279)

naspirato committed:
Mute KqpStats.SysViewClientLost (#8301)

nshestakov committed:
Fix possible write session frozing (#8075) (#8134)

nshestakov committed:
Optimize grpc for pqv0  (#8199) (#8239)

lll-phill-lll committed:
Fix null dereference (#8285)

adameat committed:
fix CORS headers (#8303)

resetius committed:
Add decimal comparison kernels (#8271)

pashandor789 committed:
[CBO] Join order hints added (#8106)

vitstn committed:
Support of functions with lang=sql (#8233)

adameat committed:
bump up viewer/query version to 3 (#8315)

va-kuznecov committed:
Refactor LOG macroses in dsproxy (#8287)

Co-authored-by: Vlad Kuznecov <va-kuznecov@nebius.com>

kungasc committed:
Report index sizes by type (#8297)

GrigoriyPA committed:
YQ-3600 fixed sessions leak in metadata service (#8312)

robdrynkin committed:
Fix space status flags logging for multiput (#8304)

StekPerepolnen committed:
describe handler added default `family` and `familyname` values (#8266)

alexvru committed:
Fix tstool file descriptor leak (#8259)

OrlovPavel committed:
grammar change (#8186)

Co-authored-by: root <p.g.orlov@tinkoff.ru>

Commit:
YDB FQ: Update github.com/ydb-platform/fq-connector-go to 0.5.6-rc.2 (#8276)

azevaykin committed:
Datashard: DISK_SPACE_EXHAUSTED error (#8318)

swalrus1 committed:
Add concurrent tiering updates to OLAP scenario test (#8262)

shnikd committed:
Take query_id to analyze coredumps (#8286)

ildar-khisambeev committed:
fix tests (#8323)

rvu1024 committed:
Serialize constraints to yson (#8211)

Nikita-Levuskin committed:
merge the additional yaml config to the main yaml config (#7136)

vitstn committed:
Support of autocommit mode for qplayer file storage (#8344)

dorooleg committed:
scheme connection has been supported (#8232)

vitstn committed:
Track sql lang function name before expand (#8333)

rekby committed:
add support skip tests, adapt tests size for end

Gazizonoki committed:
Fixed help for --null-value in import file command and refactored csv parser error handling (#3936)

Co-authored-by: Nikolay Perfilov <pnv1@yandex-team.ru>

rvu1024 committed:
AssumeConstraints callable (#8349)

Gazizonoki committed:
Moved NotExistsIsOk setting to recursive remove settings (#8355)

adameat committed:
add storage/groups latencies (#8320)

blinkov committed:
[docs] create .gitignore (#8362)

andrewstalin committed:
Added a timeout waiting for a response from the tablet (#8260)

Pseudolukian committed:
Rebase YQL CREATE (#8306)

Alek5andr-Kotov committed:
Translation of documentation (#4918)

nshestakov committed:
Use UserPool for pqv0 actors (#8316)

adameat committed:
fix segfault in autocomplete handler when redirecting to a database (#8367)

adameat committed:
refactor whiteboard info (#8366)

yumkam committed:
grace_join: redesign to improve LEFT JOIN with short left side (#8146)

rvu1024 committed:
[yt provider] Properly clear constraints in YqlRowSpec (#8350)

nikitka committed:
ci: add workflow for create github actions runner vm image (#8371)

CyberROFL committed:
Get rid of copy-pase of TChangeRecordContainer (#8352)

Commit:
ydb_topic: schedule SendImpl from OnCompressedImpl instead of calling it directly (#8372)

nshestakov committed:
Randomize order of sessions (#8359)

nshestakov committed:
Randomize semaphore name for tests (#8385)

vitstn committed:
SQL lang functions in postgis (#8374)

siarheivesialou committed:
LOGBROKER 8891 add 4 json sqs requests (#8328)

maximyurchuk committed:
Clearer comments (#8229)

rvu1024 committed:
Handle Unordered in Dq (#8357)

vitstn committed:
sql lang functions [docs] (#8378)

GrigoriyPA committed:
YQ WM fixed cleanup table retries (#8369)

alexd65536 committed:
retry requests to statistics table (#8358)

andrewstalin committed:
Statistics service metrics (#8322)

pavelvelikhov committed:
Added another cost based optimization level (#8348)

ssmike committed:
Use cpu limits from resource pool scheme objects (#8370)

nikvas0 committed:
Fix test & Sink (#8386)

va-kuznecov committed:
Disable level triggered compaction in donor VDisks (#8376)

Co-authored-by: Vlad Kuznecov <va-kuznecov@nebius.com>

rekby committed:
skip tests with break ydb

Commit:
Update github.com/ydb-platform/fq-connector-go to 0.5.6-rc.3 (#8387)

siarheivesialou committed:
LOGBROKER 8891 list dead letter source queues (#8330)

pashandor789 committed:
[KQP] Analyze OLAP supported, OLTP unsupported. (#8395)

robdrynkin committed:
Add device type in dsproxy put lwprobe (#8381)

stanislav-shchetinin committed:
Basic example for Query Service (cpp) (#7810)

pixcc committed:
Enable CMS request priorities by default (#8380)

molotkov-and committed:
[LDAP] Add flag to disable search nested groups (#8354)

ivanmorozov333 committed:
speed up merging for sparsed columns (#8365)

dorooleg committed:
s3 dqrun has been fixed (#8408)

rekby committed:
mute tests
remove prefix
fix test tenant

rekby committed:
fix mute

rekby committed:
ya.make

rekby committed:
fix ya make

rekby committed:
fix ya makes

rekby committed:
Merge branch 'main' into pg-tests

rekby committed:
fix mute format

rekby committed:
extract include

rekby committed:
use container with docker

rekby committed:
log folder

rekby committed:
Merge branch 'main' into pg-tests

rekby committed:
debug

rekby committed:
add dockerfile

rekby committed:
Merge branch 'main' into pg-tests

rekby committed:
allow docker file in root rules, add failed tests to mute

rekby committed:
add muted tests

rekby committed:
fixed style tests

rekby committed:
style, remove unneeded pg-wire option

rekby committed:
enable pg port
